src/main/java/com/ldtteam/structures/helpers/Settings.java

```diff
@@ -2,18 +2,25 @@
 
 import com.ldtteam.structurize.api.util.Shape;
 import com.ldtteam.structurize.client.gui.WindowBuildTool;
+import com.ldtteam.structurize.network.messages.LSStructureDisplayerMessage;
+
 import net.minecraft.init.Blocks;
 import net.minecraft.item.ItemStack;
 import net.minecraft.util.Mirror;
 import net.minecraft.util.Tuple;
 import net.minecraft.util.math.BlockPos;
+import net.minecraftforge.fml.common.network.ByteBufUtils;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessage;
+
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
+import io.netty.buffer.ByteBuf;
+
 /**
  * Class used to store.
  */
-public final class Settings
+public final class Settings implements IMessage
```

Comment:
Bad design.
Please make a separate message to synchronize settings properly.

Reply:
I can just remove that, it's there for from/toBytes methods and it's never used as a message

Reply:
Then remove it. This implies this is used as a message.

Reply:
Done